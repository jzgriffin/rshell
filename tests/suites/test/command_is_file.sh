mkdir test_is_file_dir
touch test_is_file_file
test -f test_is_file_dir && echo a
test -f test_is_file_file && echo b
test -f test_not_is_file_dir && echo c
test -f test_not_is_file_file && echo d
rm -rf test_is_file_dir test_is_file_file
