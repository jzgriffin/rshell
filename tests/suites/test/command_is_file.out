(False)
(True)
b
(False)
(False)
